src/xb-string.c

```diff
@@ -178,6 +178,26 @@ xb_string_search (const gchar *text, const gchar *search)
 	return FALSE;
 }
 
+/**
+ * xb_string_token_valid: (skip)
+ * @text: The potential token
+ *
+ * Checks if the token is valid.
```

Comment:
Could do with some docs to say what ‘valid’ means, and what the purpose of calling this function is (i.e. something like “to eliminate tokens which are not useful for search matching”)

Reply:
Added I think.